Content.Server/Atmos/Piping/Unary/Components/GasVentScrubberComponent.cs

```diff
@@ -42,7 +42,7 @@ public float TransferRate
 
         [ViewVariables(VVAccess.ReadWrite)]
         [DataField("maxTransferRate")]
-        public float MaxTransferRate = Atmospherics.MaxTransferRate;
+        public float MaxTransferRate = 900;
```

Comment:
Increasing the default and maximum transfer rate of scrubbers is good 👍 
Did you test if the maximum transfer rate isn't limited somewhere else in atmos code? Maybe from the pipes the scrubbers are attached to to the next pipe network? (You could probably test that with a pump set to max and also with a pasive gate).

Reply:
I do believe this is the limiting place in the code.

@juliangiebel Scrubbers operating in "Scrub" mode don't actually care about the "pipes the scrubbers are attached to" or the "next pipe network", see `Scrub()` in *GasVEntScrubberSystem.cs*:

```
        private void Scrub(float timeDelta, GasVentScrubberComponent scrubber, GasMixture? tile, PipeNode outlet)
        {
            // Cannot scrub if tile is null or air-blocked.
            if (tile == null
                || outlet.Air.Pressure >= 50 * Atmospherics.OneAtmosphere) // Cannot scrub if pressure too high.
            {
                return;
            }

            // Take a gas sample.
            var ratio = MathF.Min(1f, timeDelta * scrubber.TransferRate / tile.Volume);
            var removed = tile.RemoveRatio(ratio);

            // Nothing left to remove from the tile.
            if (MathHelper.CloseToPercent(removed.TotalMoles, 0f))
                return;

            if (scrubber.PumpDirection == ScrubberPumpDirection.Scrubbing)
            {
                _atmosphereSystem.ScrubInto(removed, outlet.Air, scrubber.FilterGases);

                // Remix the gases.
                _atmosphereSystem.Merge(tile, removed);
            }
            else if (scrubber.PumpDirection == ScrubberPumpDirection.Siphoning)
            {
                _atmosphereSystem.Merge(outlet.Air, removed);
            }
        }
```

As you can see, TransferRate multiplies the quantity of gas sampled from the tile the square is on. (Which begs the question, why not just enable "WideNet" by default?)

The other relevant line is `ScrubInto()`, which looks like this:

````
        public void ScrubInto(GasMixture mixture, GasMixture destination, IReadOnlyCollection<Gas> filterGases)
        {
            var buffer = new GasMixture(mixture.Volume){Temperature = mixture.Temperature};

            foreach (var gas in filterGases)
            {
                buffer.AdjustMoles(gas, mixture.GetMoles(gas));
                mixture.SetMoles(gas, 0f);
            }

            Merge(destination, buffer);
        }
````

There's nothing limiting it here. If this is the only concern, then I think this PR should be merged as-is.